packages/ove-app-networks/src/swagger-extensions.yaml

```diff
@@ -0,0 +1,120 @@
+tags:
+- name: "operation"
+  description: "Operations that can be performed on one or all networks"
+paths:
+  /operation/search:
+    get:
+      tags:
+      - "operation"
+      summary: "Searches for nodes or edges that match provider filters. Filters must be specified according to the OData v3.0 $filter specification."
```

Comment:
I think this is a bit misleading: it doesn't just search, it *filters* the display to hide nodes or edges that do not match a filter.